frontend/src/static/js/v_authorship.js

```diff
@@ -304,7 +320,12 @@ window.vAuthorship = {
           ? this.selectedFileTypes.reduce((a, b) => `${a}~${b}`)
           : '';
 
+      const binaryHash = this.isBinaryFilesChecked
```

Comment:
the name suggesting this should be some boolean value, even if in string

Reply:
Thanks for pointing it out. I will change it accordingly.